lib/util/secure_getenv.c

```diff
@@ -0,0 +1,9 @@
+#if !HAVE_SECURE_GETENV
+#include <stdlib.h>
+
+// Source: https://www.openwall.com/lists/musl/2019/05/28/3
+char *secure_getenv(const char *name)
```

Comment:
Just turn this into a 'static inline' function and keep it entirely in the header file?

Reply:
Yep. Much better. :)